### PR TITLE
NSFS | NC | Small fixes and Docs updates

### LINKED ISSUE #7462: NSFS | NC | reoccurring object io is invalid error messages
### Environment info
- NooBaa Version: **master**
- Platform: **minikube**

### Actual behavior
1. While running the nsfs tool (NC) I got the following error every 10 seconds - 
object io is not relevant to this tool, so we need to remove the object io semaphore sampling when using NC nsfs.
Also, the print is on debug level 0, shouldn't it be a warning/error?
```Aug-27 19:52:06.805 [nsfs/12445]    [L0] core.server.bg_services.semaphore_monitor:: semaphore_monitor: object_io is invalid undefined
Aug-27 19:52:16.810 [nsfs/12445]    [L0] core.server.bg_services.semaphore_monitor:: semaphore_monitor: object_io is invalid undefined
Aug-27 19:52:26.815 [nsfs/12445]    [L0] core.server.bg_services.semaphore_monitor:: semaphore_monitor: object_io is invalid undefined
Aug-27 19:52:36.821 [nsfs/12445]    [L0] core.server.bg_services.semaphore_monitor:: semaphore_monitor: object_io is invalid undefined
Aug-27 19:52:46.825 [nsfs/12445]    [L0] core.server.bg_services.semaphore_monitor:: semaphore_monitor: object_io is invalid undefined
Aug-27 19:52:56.830 [nsfs/12445]    [L0] core.server.bg_services.semaphore_monitor:: semaphore_monitor: object_io is invalid undefined
Aug-27 19:53:06.840 [nsfs/12445]    [L0] core.server.bg_services.semaphore_monitor:: semaphore_monitor: object_io is invalid undefined
Aug-27 19:53:16.841 [nsfs/12445]    [L0] core.server.bg_services.semaphore_monitor:: semaphore_monitor: object_io is invalid undefined
Aug-27 19:53:26.845 [nsfs/12445]    [L0] core.server.bg_services.semaphore_monitor:: semaphore_monitor: object_io is invalid undefined
Aug-27 19:53:36.852 [nsfs/12445]    [L0] core.server.bg_services.semaphore_monitor:: semaphore_monitor: object_io is invalid undefined
Aug-27 19:53:46.863 [nsfs/12445]    [L0] core.server.bg_services.semaphore_monitor:: semaphore_monitor: object_io is invalid undefined
Aug-27 19:53:56.872 [nsfs/12445]    [L0] core.server.bg_services.semaphore_monitor:: semaphore_monitor: object_io is invalid undefined
Aug-27 19:54:06.879 [nsfs/12445]    [L0] core.server.bg_services.semaphore_monitor:: semaphore_monitor: object_io is invalid undefined
Aug-27 19:54:16.887 [nsfs/12445]    [L0] core.server.bg_services.semaphore_monitor:: semaphore_monitor: object_io is invalid undefined
Aug-27 19:54:26.889 [nsfs/12445]    [L0] core.server.bg_services.semaphore_monitor:: semaphore_monitor: object_io is invalid undefined
Aug-27 19:54:36.891 [nsfs/12445]    [L0] core.server.bg_services.semaphore_monitor:: semaphore_monitor: object_io is invalid undefined
Aug-27 19:54:46.898 [nsfs/12445]    [L0] core.server.bg_services.semaphore_monitor:: semaphore_monitor: object_io is invalid undefined
Aug-27 19:54:56.901 [nsfs/12445]    [L0] core.server.bg_services.semaphore_monitor:: semaphore_monitor: object_io is invalid undefined
Aug-27 19:55:06.905 [nsfs/12445]    [L0] core.server.bg_services.semaphore_monitor:: semaphore_monitor: object_io is invalid undefined
Aug-27 19:55:16.913 [nsfs/12445]    [L0] core.server.bg_services.semaphore_monitor:: semaphore_monitor: object_io is invalid undefined
Aug-27 19:55:26.914 [nsfs/12445]    [L0] core.server.bg_services.semaphore_monitor:: semaphore_monitor: object_io is invalid undefined
Aug-27 19:55:36.916 [nsfs/12445]    [L0] core.server.bg_services.semaphore_monitor:: semaphore_monitor: object_io is invalid undefined
```

### Expected behavior
1. no irrelevant prints

### Steps to reproduce
1. Run the nsfs tool

### PR DESCRIPTION
**Pull request by romayalon**
### Explain the changes
1. Add bucket creation date to the bucket.json config file on creation, date format - "creation_date":"2023-09-26T05:56:16.252Z"
2. Stop object IO semaphore monitor prints 
3. Update NSFS non containerized docs 
 
### Issues: Fixed #xxx / Gap #xxx
1. Fixed #7462

### Testing Instructions:
1. 

- [ ] Doc added/updated
- [ ] Tests added

### CODE REVIEW

File: docs/non_containerized_NSFS.md [hunk 1]
```diff
@@ -57,23 +57,157 @@ rpm -i noobaa-core-5.14.0-1.el8.x86_64.rpm
...
+Output - 
+make_bucket: s3bucket
+```
+
+4.2. Check that the bucket configuration file was created successfully -
+```sh
+cat /tmp/noobaa_config_dir/buckets/s3bucket.json
+
+Output - 
+{ "name": "account1", "email": "account1@noobaa.io", "has_login": "false", "has_s3_access": "true", "allow_bucket_creation": "true", "access_keys": [{ "access_key": "abc", "secret_key": "123" }], "nsfs_account_config": { "uid": 0, "gid": 0, "new_buckets_path": "/tmp/fs1/", "nsfs_only": "true" } }
```
Comment (PravinRanjan10) [inline docs/non_containerized_NSFS.md:166]: Please add creation_date here in example also.

File: docs/non_containerized_NSFS.md [hunk 2]
```diff
@@ -57,23 +57,157 @@ rpm -i noobaa-core-5.14.0-1.el8.x86_64.rpm
...
+
+Output - 
+make_bucket: s3bucket
+```
+
+4.2. Check that the bucket configuration file was created successfully -
+```sh
+cat /tmp/noobaa_config_dir/buckets/s3bucket.json
+
+Output -
```
Comment (PravinRanjan10) [inline docs/non_containerized_NSFS.md:165]: Just to confirm, do we need accounts related metadata in buckets.json

**Comment by PravinRanjan10** [comment]
May be UTC instead of GMT in creation_date.
Example: 
"creation_date":"Thu Sep 21 2023 17:17:28 (UTC+0300)"

File: docs/non_containerized_NSFS.md (hunk 2 above)
Comment (romayalon) [reply docs/non_containerized_NSFS.md:165]: Do you ask if you need to add a bucket_owner/system_owner account email to the bucket json file?

**Comment by romayalon** [comment]
@PravinRanjan10 Thanks Pravin, I changed it to the following format - 
"creation_date":"2023-09-26T05:56:16.252Z"

**Review by naveenpaul1 — APPROVED** [review]

File: docs/non_containerized_NSFS.md [hunk 3]
```diff
@@ -57,19 +57,51 @@ rpm -i noobaa-core-5.14.0-1.el8.x86_64.rpm
...
+mkdir -p /tmp/fs1/
+```
+
+
 ## Run the nsfs service - 
-This systemd example script runs noobaa non containerized (currently) single mode, while /tmp/test/ is the fs root path directory.
+The systemd script runs noobaa non containerized, and requires config_root in order to find the location of the system/accounts/buckets configuration file.
+
+```sh
+systemctl start nsfs --config_root=/tmp/noobaa_config_root
```
Comment (naveenpaul1) [inline docs/non_containerized_NSFS.md:88]: @romayalon do we need to mention config_root in systemctl?

Comment (romayalon) [reply docs/non_containerized_NSFS.md:88]: we need to remove it in - https://github.com/noobaa/noobaa-core/pull/7509
i'll do that, thanks!

**Pull request merged**